app/models/edition.rb

```diff
@@ -133,7 +134,8 @@ def history_mode?
   def government
     return unless government_id
 
-    Government.find(government_id)
+    @government_repository ||= BulkData::GovernmentRepository.new
+    @government_repository.find(government_id)
```

Comment:
That won't impact the history mode scope as that runs on the class rather than an instance of the class. E.g. Edition.history_mode vs edition.government.

If this is needed in multiple places it'd make sense as a method but for now it's only needed here so I felt it was better close to usage for easy of understanding.